packages/firestore-types/index.d.ts

```diff
@@ -146,7 +151,13 @@ export class Transaction {
 
   set<T>(
     documentRef: DocumentReference<T>,
-    data: T,
+    data: Partial<T>,
+    options: SetOptions
+  ): Transaction;
+  set<T>(documentRef: DocumentReference<T>, data: T): Transaction;
+  set<T>(
+    documentRef: DocumentReference<T>,
+    data: T | Partial<T>,
     options?: SetOptions
```

Comment:
This catch-all override should not be needed in the type file.

Reply:
removed.